src/obb/server/controllers/index.cljc

```diff
@@ -0,0 +1,9 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.server.controllers.index
+  "Controller to handle the root/index request to the server")
+
+(defn handler
+  "Returns a intro to the game"
+  [request]
+  {:body "Hello World!"
+   :status 200})
```

Comment:
[personal-taste][non-blocking]

I would not call these guys controllers. We're probably not using the traditional MVC architecture and I feel less and less compelled to stick to that language. I tend to prefer routes or resources, but this is just a suggestion.